src/metabase/models/query.clj

```diff
@@ -78,21 +78,20 @@
               ;; rethrow e if updating an existing average execution time failed
               (throw e))))))
 
-;; TODO - somewhat confusing that the Ad-hoc queries here use the keys `:table-id` and `:database-id` instead of the
-;; `:database_id` and `:table_id` that come out of the Database. In the automagic dashboards code, for example, we
-;; have special utility functions to account for both possiblities. Should we fix this?
```

Comment:
removed this comment since it's not really applicable anymore now that we normalize queries when as fetch them from the DB